src/config/connection.js

```diff
@@ -0,0 +1,20 @@
+const Sequelize = require("sequelize");
+
+const dbName = process.env.DB_NAME;
+const dbUser = process.env.DB_USER;
+const dbPassword = process.env.DB_PASSWORD;
+const dbHost = process.env.DB_HOST;
+
+const dbOptions = {
+  host: dbHost,
+  user: dbUser,
```

Comment:
You do not need to add the user, database and password into the `dbOptions` as you pass them individually below on line 18.